docs/index.md

```diff
@@ -0,0 +1,329 @@
+# lolex
+
+JavaScript implementation of timing related web APIs.
+
+**Implements:**
+
+* [`setTimeout`](#settimeoutcallback-timeout-args)
+* [`clearTimeout`](#cleartimeoutid)
+* [`setImmediate`](#setimmediatecallback-args)
+* [`clearImmediate`](#clearimmediateid)
+* [`setInterval`](#setintervalcallback-timeout-args)
+* [`clearInterval`](#clearintervalid)
+* [`requestAnimationFrame`](#requestanimationframecallback)
+* [`cancelAnimationFrame`](#cancelanimationframeid)
+* [`Date`](#date)
+* [`nextTick`](#nexttickcallback-args)
+* [`queueMicrotask`](#queuemicrotaskcallback)
+* [`performance.now`](#performancenow)
+
+along with a clock instance that controls the flow of time.
+**lolex** also provides a [`Date`](#date) implementation that gets its time from the clock.
+
+**lolex** can be used to simulate passing time in automated tests and other
+situations where you want the scheduling semantics, but don't want to actually
+wait (however, from version 2.0 **lolex** supports those of you who would like to wait too).
+
+**lolex** is a part of [Sinon.JS](https://github.com/sinonjs/sinon.js).
+
+## Installation
+
+```shell
+npm install lolex --save-dev
+```
+
+You are always free to [build it yourself](https://github.com/sinonjs/lolex/blob/master/package.json#L22), of course.
+
```

Comment:
Please let me know your thoughts about the following refactoring:
## Usage

Use **lolex** in Node.js and browser environments.

In the Node.js environment, **lolex** provides the `nextTick` implementation to synchronize the testing environment clock with the system clock and a `process.hrtime` shim that works with the clock.

In the browser, **lolex** provides the `performance` implementation to synchronize the clock in your testing environment with the system clock. Use the pre-built version available in the repo and the npm package.

When using npm, you need only to reference **lolex** in your <script> tags.
```html
<script type="text/javascript" src="https://unpkg.com/lolex@1.4.0/src/lolex.js">
...
</script>
```
The following code sample creates a new clock, schedules an event with the [`setTimeout function`](#setTimeout), and moves the clock ahead using the [`tick`](#tick) method.

 ```js
var lolex = require("lolex");
 var clock = lolex.createClock();
 clock.setTimeout(function () {
    console.log("The poblano is a mild chili pepper originating in the state of Puebla, Mexico.");
}, 15);
 // ...
 clock.tick(15);
```
**NOTE:** In the browser distribution, a global lolex clock is available by default.

After the last line is executed, an interesting fact about the [Poblano](http://en.wikipedia.org/wiki/Poblano) pepper is displayed on the screen. If you want to simulate asynchronous behavior, you have to use your imagination when calling the various functions.

In addition to the [`tick`](#tick) method, the [`next`](#next), [`runAll`](#runall), [`runToFrame`](#runtoframe), and [`runToLast`](#runtolast) methods are available to advance the clock. See the [clock API Reference](#clock-api) for more details.

Reply:
@rjseymour66 
> In the Node.js environment, **lolex** provides the `nextTick` implementation to synchronize the testing environment clock with the system clock ...

This is not correct. The [`nextTick`](https://nodejs.org/api/process.html#process_process_nexttick_callback_args) method schedules the given callback for the next iteration of the event loop.

> In the browser, **lolex** provides the `performance` implementation to synchronize the clock in your testing environment with the system clock.

Also wrong. Please have have look [here](https://developer.mozilla.org/en-US/docs/Web/API/Performance) on what is the `performance` API meant for.

Any other thoughts on this?
@benjamingr @SimenB

Reply:
@rjseymour66 the rest is fine BTW :+1:

Reply:
Think i just commented in the wrong area, so excuse the duplication:

I made changes to the nextTick and performance methods